nix/default.nix

```diff
@@ -143,54 +208,21 @@ let
 		# Like `buildPackageSpec` but only returns the single selected package.
 		buildPackage = name: buildPackageConstraint { inherit name; };
 
-		# build a nix derivation from a (local) opam library, i.e. one not in the official repositories
+		buildOpamPackages = packages: drvAttrs: buildOpamPackages packages drvAttrs;
```

Comment:
nope, because it's not in a `rec` attrset. But it's unnecessary and confusing, replaced with a simple `inherit` in 23ab7f734f2f80c6214d10f157c1128e6e190b71